akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -632,10 +633,28 @@ Host: example.com
       val clientSettings = ConnectionPoolSettings(system).withConnectionSettings(ClientConnectionSettings(system).withSocketOptions(SO.ReceiveBufferSize(serverToClientNetworkBufferSize) :: Nil))
 
       val serverConnectionContext = ExampleHttpContexts.exampleServerContext
-      // Disable hostname verification as ExampleHttpContexts.exampleClientContext sets hostname as akka.example.org
-      val sslConfigSettings = SSLConfigSettings().withLoose(SSLLooseConfig().withDisableHostnameVerification(true))
-      val sslConfig = AkkaSSLConfig().withSettings(sslConfigSettings)
-      val clientConnectionContext = ConnectionContext.https(ExampleHttpContexts.exampleClientContext.sslContext, Some(sslConfig))
+
+      // Create SSLEngine without hostname verification as ExampleHttpContexts.exampleClientContext sets hostname as akka.example.org
+      // Create SSLEngine without hostname verification as ExampleHttpContexts.exampleClientContext sets hostname as akka.example.org
+      val context = {
```

Comment:
Can we update `ExampleHttpContexts.exampleClientContext` instead to make that configurable (or provide a different way to make it look as if we were connecting to akka.example.org, e.g. using a custom ClientTransport)?